docs/devel/local-cluster/docker.md

```diff
@@ -225,6 +214,10 @@ On OS X, since docker is running inside a VM, run the following command instead:
 docker-machine ssh `docker-machine active` curl $ip
 ```
 
+## Deploy a DNS
+
+Read [documentation for manually deploying a DNS](http://kubernetes.io/docs/getting-started-guides/docker-multinode/#deploy-dns-manually-for-v12x) for instructions.
```

Comment:
What is the general policy for cross-linking? I assume over time these links might break.